CHANGELOG.md

```diff
@@ -32,6 +34,7 @@
 ### Breaking Changes
 - We needed to upgrade argilla-server image version from `argilla-server:v1.26.0` to `argilla-server:v1.29.0` to maintain compatibility.
   - Note: We also updated our elasticsearch argilla backend to `8.12.2`
+- The default value of `threshold` in the `DocumentIndexRetriever` is now `0.0`. This accommodates fusion scoring for searches over hybrid indexes.
```

Comment:
move to unreleased. Maybe also add that it was `0.5` previously, so people can easily change it based on the changelog